uwnet/datasets.py

```diff
@@ -128,6 +128,50 @@ def scale(self):
         return valmap(lambda x: x.max(), std)
 
 
+class XRTimeSeriesEta(XRTimeSeries):
+    """Same as XRTimeSeries, but only trained on a specific eta value.
+    """
+
+    def __init__(self, data, eta):
+        """
+        Parameters
+        ----------
+        data : xr.DataArray
+            An input dataset. This dataset must contain at least some variables
+            with all of the dimensions ['time' , 'z', 'x', 'y'].
+        eta : int
+            The value of eta to train on.
+        """
+        self.eta = eta
```

Comment:
Let's try not to duplicate code from `XRTimeSeries`. This code should work:
```
super(XRTimeSeriesEta, self).__init__(data, time_length=2)
self.eta = eta
self.setup_indices()
```

Reply:
Thats better. The last line is actually not necessary because self.setup_indices() is called in the __init__ of XRTimeSeries

Reply:
sure. sounds good